core/includes/error.tpl

```diff
@@ -0,0 +1,351 @@
+                    {foreach from=$FRAMES item=frame}
+                        <button class="tablinks item" id="button-{$frame['number']}" onclick="openFrame({$frame['number']})">
+                            <h5>Frame #{$frame['number']}</h5>
+                            <sub>{$frame['file']}:{$frame['line']}</sub>
+                        </button>
+                    {/foreach}
+                </div>
+
+                {foreach from=$FRAMES item=frame}
+                    <div id="frame-{$frame['number']}" class="tabcontent">
+                        <h5 style="overflow: scroll !important;">{$frame['file']}</h5>
+
+                        {if $frame['code'] != ''}
+                            <pre data-line="{$frame['highlight_line']}"
+                                 data-start="{$frame['start_line']}"><code class="language-php line-numbers">{$frame['code']}</code></pre>
+                        {else}
+                            <pre class="text-center">Cannot read file.</pre>
+                        {/if}
+                    </div>
+                {/foreach}
+            </div>
+
+            <div class="ui bottom attached tab segment active" id="sql" data-tab="sql" style="border: 1px solid #d4d4d5; border-radius: 3px;">
+                <div class="ui tab secondary vertical menu left floated" id="sql-tablinks-container" style="max-height: 950px; overflow-y: scroll;">
+                    {foreach from=$ERROR_SQL_STACK item=$stack}
+                        <button class="sql-tablinks item" id="sql-button-{$stack['number']}" onclick="openSqlFrame({$stack['number']})">
+                            <h5>Query #{$stack['number']}</h5>
+                            <sub>{$stack['frame']['file']}:{$stack['frame']['line']}</sub>
+                        </button>
+                    {/foreach}
+                </div>
+
+                {foreach from=$ERROR_SQL_STACK item=$stack}
+                    <div id="sql-frame-{$stack['number']}" class="sql-tabcontent">
+                        <h5 style="overflow: scroll !important;">{$stack['frame']['file']}</h5>
+                        {$stack['sql_query']}
+
+                        {if $stack['frame']['code'] != ''}
+                            <pre data-line="{$stack['frame']['highlight_line']}"
+                                data-start="{$stack['frame']['start_line']}"><code class="language-php line-numbers">{$stack['frame']['code']}</code></pre>
+                        {else}
+                            <pre class="text-center">Cannot read file.</pre>
+                        {/if}
+                    </div>
+                {/foreach}
+            </div>
+        </div>
+
+        {/if}
+
+    </div>
+
+    {if $CAN_GENERATE_DEBUG}
+        <div class="ui modal tiny" id="debug_link_modal">
+            <i class="close icon"></i>
+            <div class="header">
+                Generate a debug link
```

Comment:
Should we be looking to translate all terms in this file? Maybe in the future?

Reply:
Good catch